tensorbay/sensor/intrinsics.py

```diff
@@ -489,6 +768,31 @@ def project(self, point: Sequence[float], is_fisheye: bool = False) -> Vector2D:
         Returns:
             The coordinates on the pixel plane where the point is projected to.
 
+        Examples:
+            Project a point with 2 dimensions.
+
+            >>> coordinates = camera_intrinsics.project((1, 2))
```

Comment:
Since the coordinates are no longer used in the following examples, it would be better to use the following style:
```lang=python
>>> camera_intrinsics.project((1, 2))
Vector2D(137.0, 510.0)
```